server/src/main/java/org/elasticsearch/action/update/UpdateRequest.java

```diff
@@ -120,7 +120,26 @@
     public UpdateRequest() {
 
     }
+    
+    /**
+     * Constructor taking the parameters required to identify the document
+     * @param index The index containing the doc
+     * @param id The unique id of the document in the index
+     */
+    public UpdateRequest(String index, String id) {
+        super(index);
+        this.type = "_doc";
+        this.id = id;
```

Comment:
Would be nice to standardise on if we use "_doc" or MapperService.SINGLE_MAPPING_NAME in code.
I can see pros and cons to each approach. Using "_doc" reduces dependencies and will make detangling HLRC request/response pairs from server classes easier. Using a constant like "MapperService.SINGLE_MAPPING_NAME makes tracing usage easier.